src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -277,90 +320,110 @@ const WorkspaceDashboard = _.flow(
-              disabled: busy,
-              onClick: () => deleteTag(tag),
-              style: { marginLeft: '0.25rem', verticalAlign: 'middle', display: 'inline-block' }
-            }, [icon('times', { size: 14 })])
-          ])
-        }, tagsList),
-        !!tagsList && _.isEmpty(tagsList) && i(['No tags yet'])
+      h(RightBoxSection, {
+        title: 'Owners',
+        initialOpenState: ownersPanelOpen,
+        onClick: () => setOwnersPanelOpen(!ownersPanelOpen)
+      }, [
+        div({ style: { margin: '0.5rem' } },
+          _.map(email => {
+            return div({ key: email, style: { overflow: 'hidden', textOverflow: 'ellipsis', marginBottom: '0.5rem' } }, [
+              h(Link, { href: `mailto:${email}` }, [email])
+            ])
+          }, owners))
       ]),
-      !_.isEmpty(authorizationDomain) && h(Fragment, [
-        div({ style: Style.dashboard.header }, ['Authorization Domain']),
-        div({ style: { marginBottom: '0.5rem' } }, [
+      !_.isEmpty(authorizationDomain) && h(RightBoxSection, {
+        title: 'Authorization domain',
+        initialOpenState: authDomainPanelOpen,
+        onClick: () => setAuthDomainPanelOpen(!authDomainPanelOpen)
+      }, [
+        div({ style: { margin: '0.5rem 0.5rem 1rem 0.5rem' } }, [
           'Collaborators must be a member of all of these ',
           h(Link, {
             href: Nav.getLink('groups'),
             ...Utils.newTabLinkProps
           }, 'groups'),
           ' to access this workspace.'
         ]),
-        ..._.map(({ membersGroupName }) => div({ style: styles.authDomain }, [membersGroupName]), authorizationDomain)
-      ]),
-      div({ style: { margin: '1.5rem 0 1rem 0', borderBottom: `1px solid ${colors.dark(0.55)}` } }),
-      div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
-        'Google Bucket'
-      ]),
-      div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
-        div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Name:']),
-        h(TooltipCell, { style: { marginRight: '0.5rem' } }, [bucketName]),
-        h(ClipboardButton, { text: bucketName, style: { marginLeft: '0.25rem' } })
-      ]),
-      div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
-        div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Location:']),
-        bucketLocation ? h(Fragment, [
-          div({ style: { marginRight: '0.5rem' } }, [flag]),
-          regionDescription
-        ]) : 'Loading...'
+        ..._.map(({ membersGroupName }) => div({ style: { margin: '0.5rem', fontWeight: 500 } }, [membersGroupName]), authorizationDomain)
       ]),
-      h(Link, {
-        ...Utils.newTabLinkProps,
-        href: bucketBrowserUrl(bucketName)
-      }, ['Open in browser', icon('pop-out', { size: 12, style: { marginLeft: '0.25rem' } })])
+      h(RightBoxSection, {
+        title: ['Tags',
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
```

Comment:
When you click on the info icon, the section expands/collapses. I think you need to stop the event propagation or the info icon needs to go somewhere else.

Reply:
This was on my docket to fix before undrafting :) Fixed in 560038c10892851e8a103c3d8974cc11a6a87155

Reply:
Sure, no problem. I just wanted to start getting a handle on the redesign as I'm also going to be working in this file momentarily.